### PR TITLE
test(sidebar, settingsrow,showeditpod): update cypress with new approach

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Apply new `url` approach in tests for:
`sidebar` (timing: was `1:18` -> `00:22` ),
`settingsRow` (timing: `00:53` was -> `00:10` ),
`showEditPod` (timig: `2:04` was  -> `00:42`)

### Current behaviour

Currently we are passing all parameters via `knobs`.

### Checklist

- [x] Commits follow our style guide
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

File: cypress/features/regression/showEditPod.feature [hunk 1]
```diff
@@ -1,138 +1,135 @@
...
-      | mp150ú¿¡üßä                  |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | saveText                     | nameOfObject             |
+      | mp150ú¿¡üßä                  | saveTextOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | saveTextSpecialCharacter |
 
   @positive
   Scenario: Enable saving checkbox for a Show Edit Pod component
-    When I check saving checkbox
+    When I open default "ShowEditPod" component in noIFrame with "showEditPod" json from "commonComponents" using "save" object name
```
Comment (DlgSHi) [inline cypress/features/regression/showEditPod.feature:83]: `Given` - `When` - `Then`

File: cypress/features/regression/showEditPod.feature [hunk 2]
```diff
@@ -1,138 +1,135 @@
...
       And I click edit Show Edit Pod component
     Then Show Edit Pod component has saving property
 
   @positive
-  Scenario: Enable and disable saving checkbox for a Show Edit Pod component
-    Given I check saving checkbox
-    When I uncheck saving checkbox
+  Scenario: Disable saving checkbox for a Show Edit Pod component
+    When I open default "ShowEditPod" component in noIFrame with "showEditPod" json from "commonComponents" using "saveFalse" object name
```
Comment (DlgSHi) [inline cypress/features/regression/showEditPod.feature:89]: `Given` - `When` - `Then`

File: cypress/features/regression/sidebarWithButton.feature [hunk 3]
```diff
@@ -7,11 +7,13 @@ Feature: Sidebar With Button component
 
   @positive
   Scenario: Open Sidebar With Button
+    # commented because of BDD default scenario Given - When - Then
```
Comment (DlgSHi) [inline cypress/features/regression/sidebarWithButton.feature:10]: I think we should delete this `.feature` file.

File: cypress/fixtures/commonComponents/showEditPod.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    "buttonAlign": "right"
+  },
+  "cancelTextOtherLanguage": {
+    "cancelText": "mp150ú¿¡üßä"
+  },
+  "cancelTextSpecialCharacter": {
+    "cancelText": "!@#$%^*()_+-=~[];:.,?{}&\"'<>"
+  },
+  "defualt": {
+
```
Comment (DlgSHi) [inline cypress/fixtures/commonComponents/showEditPod.json:21]: empty line?

File: cypress/fixtures/commonComponents/showEditPod.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  "cancelFalse": {
+    "cancel": false
+  },
+  "deleteTextOtherLanguage": {
+    "deleteText": "mp150ú¿¡üßä"
+  },
+  "deleteTextSpecialCharacter": {
+    "deleteText": "!@#$%^*()_+-=~[];:.,?{}&\"'<>"
+  },
+  "save": {
```
Comment (DlgSHi) [inline cypress/fixtures/commonComponents/showEditPod.json:35]:
```suggestion
  "saving": {
```

File: cypress/fixtures/commonComponents/showEditPod.json [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    "saving": true
+  },
+  "saveFalse": {
```
Comment (DlgSHi) [inline cypress/fixtures/commonComponents/showEditPod.json:38]:
```suggestion
  "savingFalse": {
```

**Review by DlgSHi — COMMENTED** [review]
1. Add time please in the PR description
2. Some tests failed

File: cypress/features/regression/showEditPod.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/showEditPod.feature:83]: I won't change it. Important is step `When` and it specifies step `Then`

File: cypress/features/regression/showEditPod.feature (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/features/regression/showEditPod.feature:89]: I won't change it. Important is step `When` and it specifies step `Then`

**Review by DlgSHi — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Pull request merged**